lib/modules/manager/vendir/__fixtures__/git-contents.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+directories:
+- path: vendor
+  contents:
+  - path: custom-repo-custom-version
```

Comment:
please join fixtures, so we've less files and less reads. we usually have one big fixture with all supported valid values and a single test which verified it. so adding new supported values needs less test code changes.